packages/cli/src/ui/commands/setupGithubCommand.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * @license
+ * Copyright 2025 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { execSync } from 'child_process';
+
+import {
+    CommandKind,
+    SlashCommand,
+    SlashCommandActionReturn,
+} from './types.js';
+
+export const setupGithubCommand: SlashCommand = {
+    name: 'setup-github',
+    description: 'Set up GitHub Actions using the Gemini CLI',
+    kind: CommandKind.BUILT_IN,
+    action: (): SlashCommandActionReturn => {
+        const gitRootRepo = execSync('git rev-parse --show-toplevel', {
+            encoding: 'utf-8',
+        }).trim();
+
+        if (!gitRootRepo) {
+            throw new Error('Unable to determine the Git root directory.');
+        }
+
+        const workflowBaseUrl =
+            `https://raw.githubusercontent.com/google-gemini/gemini-cli-action/main/.github/workflows`;
+
+        const workflows = [
```

Comment:
If one of these workflows aren't available we should fail gracefully and abort the setup imo

Reply:
Updated.